src/components/freelancer/project/projectDetailCard.tsx

```diff
@@ -11,14 +11,14 @@ import {
 import { Badge } from '@/components/ui/badge';
 
 export interface ProjectDetailCardProps {
```

Comment:
please use similar fallback types for similar fields, domain  and skills should have similar type fallback as they have same main types
same for status and other string types fields